using_images/xpaas_images/eap.adoc

```diff
@@ -1,119 +1,110 @@
 
-* The JBoss EAP Management Console is not available link:#Managing-OpenShift-JBoss-EAP-xPaaS-Images[to manage OpenShift JBoss EAP xPaaS images].
-* The JBoss EAP Management CLI is only bound locally. This means that you can only access the Management CLI of a container from within the pod.
-* Domain mode is not supported. OpenShift controls the creation and distribution of applications in the containers.
-* The default root page is disabled. You may want to deploy your own application to the root context (as *_ROOT.war_*).
-* A-MQ is supported for inter-pod and remote messaging. HornetQ is only supported for intra-Pod messaging and only enabled in the absence of A-MQ.
+== Comparison Between the xPaaS JBoss EAP 6.4 and 7.0 Beta Images
 
-[[Managing-OpenShift-JBoss-EAP-xPaaS-Images]]
-=== Managing OpenShift JBoss EAP xPaaS Images
+Red Hat offers two xPaaS EAP images for use with OpenShift. The first is based on JBoss EAP 6.4 and the second is based on JBoss EAP 7 Beta. There are several differences between the two images:
 
-A major difference in managing an OpenShift JBoss EAP xPaaS image is that there is no Management Console exposed for the JBoss EAP installation inside the image. Because images are intended to be immutable, with modifications being written to a non-persistent file system, the Management Console is not exposed.
+**JBoss Web is replaced by  Undertow**
 
-However, the JBoss EAP Management CLI (*_EAP_HOME/bin/jboss-cli.sh_*) is still
-accessible from within the container for troubleshooting purposes. First open a
-remote shell session to the running pod:
+. The xPaaS JBoss EAP 6.4 image uses JBoss Web.
+. The xPaas JBoss EAP 7 Beta image uses Undertow instead of JBoss Web. This change only affects users implementing custom JBoss Web Valves in their applications. Affected users must refer to the Red Hat JBoss EAP 7 Beta documentation for details about JBoss EAP Web Valve handlers.
 
-----
-$ oc rsh <pod_name>
-----
+**HornetQ is replaced by Artemis**
 
-Then run the following from the remote shell session to launch the JBoss EAP
-Management CLI:
+. The EAP 6.4 image only uses HornetQ for intra-pod messaging when A-MQ is absent.
+. The EAP 7 Beta image uses Artemis instead of HornetQ. This change resulted in renaming the `*HORNETQ_QUEUES*` and `*HORNETQ_TOPICS*` environment variables to `MQ_QUEUES` and `MQ_TOPICS` respectively. For complete instructions to deal with migrating applications from JBoss EAP 6.4 to 7 Beta, see the JBoss EAP 7 Beta Migration Guide.
+. The EAP 7 Beta image uses Artemis instead of HornetQ. This change resulted in renaming the `*HORNETQ_QUEUES*` and `*HORNETQ_TOPICS*` environment variables to `*MQ_QUEUES*` and `*MQ_TOPICS*` respectively. For complete instructions to deal with migrating applications from JBoss EAP 6.4 to 7 Beta, see the [JBoss EAP 7 Beta Migration Guide].
```

Comment:
what do the square brackets around the guide name indicate here? Could you make that a link to the guide instead?